docs/typescript.md

```diff
@@ -35,3 +35,7 @@ const [num, setNum] = useAtom(primitiveNumAtom)
 const [num] = useAtom(readOnlyNumAtom)
 const [, setNum] = useAtom(writeOnlyNumAtom)
 ```
+
+#### Version requirement
```

Comment:
```suggestion
## Version requirement
```

This would be better.

I don't mind putting this section before `## Notes` but it's up to you.

Reply:
I updated it on commit https://github.com/pmndrs/jotai/pull/152/commits/388994a3727656ca0191498fecfe8850c7a5f433.